blaze/avatar.js

```diff
@@ -18,7 +18,7 @@ Template.avatar.helpers({
 
   imageUrl: function () {
     var user = this.user ? this.user : Meteor.users.findOne(this.userId);
```

Comment:
If user becomes optional then it's worth avoiding the findOne() when there's no userId specified